doc/source/templates/04_finetuning_llms_with_deepspeed/README.md

```diff
@@ -110,6 +105,34 @@ This script was tested across three model sizes on the following cluster configu
 | 70B        | `meta-llama/Llama-2-70b-hf`  | 8                     | 32x A10G (24G) | ~190 min.             |
 
 
+To launch a successful finetuning you can use the following command:
+
+```
+./run_llama_ft.sh --size=7b --num-epochs=10
+```
+
+Note that the learning rate schedule scales with the number of epochs.
+Hence, training with ```./run_llama_7b.sh --num-epochs=2``` yields different results after training for 1 epoch then ```./run_llama_7b.sh --num-epochs=1```.
```

Comment:
wait the script does not allow you to specify `--num-epochs`.Let's keep it simple and hardcode to 1 epoch total.

Reply:
The point of this shell script is to set a hello-world entry point.

Reply:
No need to highligh the complexities of learning rate.